examples/r_wine/train.R

```diff
@@ -26,7 +27,7 @@ lambda <- mlflow_param("lambda", 0.5, "numeric")
 
 with(mlflow_start_run(), {
     model <- glmnet(train_x, train_y, alpha = alpha, lambda = lambda, family= "gaussian", standardize = FALSE)
-    predictor <- crate(~ glmnet::predict.glmnet(model, as.matrix(.x)), model)
+    predictor <- crate(~ glmnet::predict.glmnet(!!model, as.matrix(.x)), !!model)
```

Comment:
what are the '!!' for?

Reply:
`!!` is needed by `crate` to capture local state, in this case `model`
https://www.rdocumentation.org/packages/carrier/versions/0.1.0/topics/crate